src/euphorie/client/settings.py

```diff
@@ -197,6 +197,15 @@ def getContent(self):
         return user
 
     def initiateRequest(self, account, login):
+        flash = IStatusMessage(self.request).addStatusMessage
```

Comment:
Really minor: I would use plone api.portal.show_message

Reply:
thanks, I just re-used the existing code and did not care about cleaning up